### DISCUSSION TITLE
id_quest en cas de groupe et questionnaires multiples

### DISCUSSION THREAD

**Discussion by JMPivette** [JMPivette/evavelo#17] [Q&A] [answered]
@VeloTerritoires

Je n'ai pas l'exemple dans le fichier type. Peux-tu me confirmer que si j'ai plusieurs questionnaires sur un même groupe, alors les données ressembleront à cet exemple: (Groupe de 3 cycliste ayant rempli 2 questionnaires:

Comptages_manuels:
- id_quest: 106a 
Enquete_saisies:
- id_quest: 106a-1
- id_quest: 106a-2

Le suffixe est bien systématiquement un tiret suivi de valeurs numériques?

**Comment by SteffMangin** [comment, marked as answer]
C'est la préconisation en effet. Pourquoi, cela a une importance pour toi ?

**Reply by JMPivette** [reply]
Quand je vérifie la présence des id_quest dans  `comptage`et dans `enquete`, je dois enlever le suffixe.
J'ai 2 manière de le faire:
- enlever tout ce qui fini par `-` puis des valeurs numériques:
- enlever tout ce qu'il y a après un `-`

Si on remplacait `106a-2` par `106a-b`, ma première méthode ne fonctionnerait pas.
Si on ajoute un `-`dans l'id du questionnaire (sans suffixe), ma deuxième méthode ne marche pas: `10-6a`

La question est savoir ce qu'elle est l'erreur la plus probable.

**Reply by SteffMangin** [reply]
Est-ce que ce ne serait pas plus simple d'avoir dans le fichier standard, 2 colonnes : 1 pour le id_quest global et 1 pour le id_quest_detail en cas de réponses multiples dans un groupe ?

**Reply by SteffMangin** [reply]
@JMPivette  est-ce que tu avais vu ma réponse ? J'avais oublié de te notifier... Désolée

**Reply by JMPivette** [reply]
J'avais vu ta réponse, mais je n'avais pas encore eu le temps de répondre.
Pas vraiment de préférence de mon côté. J'ai juste un peu peur qu'en créant 2 colonnes, cela rende le document moins lisible et entraîne des erreurs de saisie.

**Reply by SteffMangin** [reply]
@JMPivette 
ok dans ce cas, j'imagine que tu es parti sur le principe du tiret et du chiffre comme préconisé dans le guide.
J'insisterai donc lourdement dans le fichier et dans le catalogue de variable pour que ce format soit respecté.
